patch-upstream.sh

```diff
@@ -1,7 +1,7 @@
 cp -rf adot/* opentelemetry-lambda/
 cd opentelemetry-lambda/collector
 # replace collector with AOC
-go mod edit -replace github.com/open-telemetry/opentelemetry-lambda/collector/lambdacomponents=github.com/aws-observability/aws-otel-collector/pkg/lambdacomponents@v0.11.0
+go mod edit -replace github.com/open-telemetry/opentelemetry-lambda/collector/lambdacomponents=github.com/aws-observability/aws-otel-collector/pkg/lambdacomponents@v0.0.0-20210708212027-05f20388cb49
```

Comment:
links to this commit: https://github.com/aws-observability/aws-otel-collector/commit/05f20388cb494af05c182b59259a9304b350be15#diff-8cd2d7a3f2e077d631acd373c7ce2d806f1c4ff2d7da21a36d6e537ccc1b13e3